ivcheck.py

```diff
@@ -127,14 +127,23 @@ def __init__(self, args):
         if location in self.config['waits']:
             await asyncio.sleep(self.config['waits'][location])
 
-    async def swipe(self, location, duration):
-        await self.p.swipe(
-            self.config['locations'][location][0],
-            self.config['locations'][location][1],
-            self.config['locations'][location][0],
-            self.config['locations'][location][1],
-            duration
-        )
+    async def swipe(self, location, duration, reverse=False):
+        if len(self.config['locations'][location]) == 2:
```

Comment:
Is this if block for a tap&hold/long-tap behaviour (the one we simulate with swipes because of the lack of embedded support)?